packages/libs/wrapped-keys/src/lib/Lit.d.ts

```diff
@@ -0,0 +1,362 @@
+    }): Promise<string>;
+    /**
+     * Call a smart contract
+     * @function callContract
+     * @param {Object} params
+     * @param {string} params.chain The name of the chain to use.  Check out the lit docs "Supported Blockchains" page to find the name.  For example, "ethereum"
+     * @param {string} params.txn The RLP Encoded txn, as a hex string
+     * @returns {Promise<string>} The response from calling the contract
+     */
+    function callContract({ chain, txn }: { chain: string; txn: string }): Promise<string>;
+    /**
+     * Convert a Uint8Array to a string.  This is a re-export of this function: https://www.npmjs.com/package/uint8arrays#tostringarray-encoding--utf8
+     * @function uint8arrayToString
+     * @param {Uint8Array} array The Uint8Array to convert
+     * @param {string} encoding The encoding to use.  Defaults to "utf8"
+     * @returns {string} The string representation of the Uint8Array
+     */
+    function uint8arrayToString(...args: any[]): string;
+    /**
+     * Convert a string to a Uint8Array.  This is a re-export of this function: https://www.npmjs.com/package/uint8arrays#fromstringstring-encoding--utf8
+     * @function uint8arrayFromString
+     * @param {string} string The string to convert
+     * @param {string} encoding The encoding to use.  Defaults to "utf8"
+     * @returns {Uint8Array} The Uint8Array representation of the string
+     */
+    function uint8arrayFromString(...args: any[]): Uint8Array;
+    function aesDecrypt({ symmetricKey, ciphertext }: { symmetricKey: any; ciphertext: any }): any;
+    /**
+     * Claim a key through a key identifier, the result of the claim will be added to `claim_id`
+     * under the `keyId` given.
+     * @param {Object} params
+     * @param {string} params.keyId user id of the claim
+     */
+    function claimKey({ keyId }: { keyId: string }): any;
+    /**
+     * Broadcast a message to all connected clients and collect their responses
+     * @param {string} name  The name of the broadcast
+     * @param {string} value  The value to broadcast
+     * @returns {string} The collected responses as a json array
+     */
+    function broadcastAndCollect({ name, value }: string): string;
+    /**
+         * Decrypt and combine the provided
+         * @param {string} accessControlConditions The access control conditions
+         * @param {string} ciphertext The ciphertext to decrypt
+         * @param {string} dataToEncryptHash The hash of the data to <encrypt />
+         @ @param {string} authSig The auth signature
+         * @param {string} chain The chain
+         * @returns {string} The combined data
+         */
+    function decryptAndCombine({
+      accessControlConditions,
+      ciphertext,
+      dataToEncryptHash,
+      authSig,
+      chain,
+    }: {
+      accessControlConditions: unknown[];
+      ciphertext: string;
+      dataToEncryptHash: string;
+      chain: string;
+      authSig: any;
+    }): string;
```

Comment:
@MaximusHaximus I copied this file from the link you shared but the type for the object was `string`, so had to correct it, but it's still wrong in the Starter Kit - I can update there as well, assuming this wasn't intended?